sources/ecommerce-app/src/redux/reducers/productsReducer.js

```diff
@@ -122,7 +123,7 @@ export default function productReducer(state = initialState, action) {
 
     case CHANGE_LOCALE:
     case CHANGE_CURRENCY: {
-      Cookies.set(type === CHANGE_LOCALE ? 'locale' : 'currency', payload.value);
+      Cookies.set(type === CHANGE_LOCALE ? `crafter-engine-locale-${siteName}` : 'currency', payload.value);
```

Comment:
This is a side effect and should be on an epic

Reply:
Also, we should refactor that string creation to a function that can be called for read and write